draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -389,11 +389,11 @@
                 is established by the answerer) from the answerer before the offerer receives the SDP answer.
             </t>
             <t>
-                NOTE: If the offerer receives a DTLS ClientHello message, and a DTLS association is established,
+                If the offerer receives a DTLS ClientHello message, and a DTLS association is established,
                 before the offerer receives the SDP Answer carrying the fingerprint associated with the DTLS
-                association, any data received on the DTLS association before the fingerprint is received is to be
+                association, any data received on the DTLS association before the fingerprint MUST be
                 considered coming from an unverified source. The processing of such data is outside the scope of
-                this document.
+                this document. The offerer MUST NOT send any data before the DTLS assocation has been established.

```

Comment:
Are you saying that no data can be send before DTLS association is established or that no data should be sent before answer is received and remote party is verified? Sending data before DTLS association is established makes no sense,

What does it mean "MUST NOT send any data"? This is really ambiguous. Do you mean everything or actual playable media or content data? Can, for instance, SCTP association handshake proceed before DTLS association is verified or not?

I think, if we are leaving data handling up to the application sending data should also be up to the application. So, I think saying "The processing of such data or sending data to unverified remote party is outside the scope of this document."